src/ansys/mechanical/core/run.py

```diff
@@ -245,3 +245,121 @@ def cli(
 
     if private_appdata:
         profile.cleanup()
+
+
+@click.command()
+@click.help_option("--help", "-h")
+@click.option(
+    "-r",
+    "--revision",
+    default=None,
+    type=int,
+    help='Ansys Revision number, e.g. "241" or "232". If none is specified\
+, uses the default from ansys-tools-path',
+)
+@click.argument("args", nargs=-1)
+def update_environment(revision: int, args):
+    """CLI tool to load the environment required to run in Linux.
+
+    Parameters
+    ----------
+    revision : int
+        The Ansys Revision number.
+
+    USAGE:
+    The following example demonstrates the main use of this tool:
+        $ mechanical-env -r 232 -- python examples/basic/valve.py
+        $ mechanical-env -r 232 -- make -C doc html
+    """
+    #  Should not update env variables in Windows
+    if os.name == "nt":
+        print("This feature is not available in Windows !")
+        return True
+
+    # Gets the revision number
+    if not revision:
+        exe, version = atp.find_mechanical()
+    else:
+        exe, version = atp.find_mechanical(version=revision)
+    version = int(version * 10)
+
+    env = os.environ.copy()
+
+    # workbench (mechanical) installation directory
+    env["DS_INSTALL_DIR"] = os.path.dirname(exe)
+    env[f"AWP_ROOT{version}"] = f'{env.get("DS_INSTALL_DIR")}/..'
+
+    # Env vars used by workbench (mechanical) code
```

Comment:
comare the .workbench_lite between 232 and 241, there might be some differences, for which you'll have to respect by using the `revision` argument

Reply:
Updated